calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -72,7 +72,7 @@ private void OnPropertyChanged([CallerMemberName] string text = null)
         public void ShowNumberOnDisplay(string textToDisplay)
         {
             DisplayContent += textToDisplay;
-            var removedLeadingZeros = decimal.Parse(DisplayContent);
+            var removedLeadingZeros = DisplayContent.TrimStart('0');
```

Comment:
While it's true that not parsing an arbitrarly long string into a decimal will prevent the Overflow Exception,
trimming all leading 0's **including** the last 0 causes the display to show no character. Which is undesired behavior.
The display should display at least the "0" character. However, "000" should be trimmed to "0", not an empty string "".

![mtF5ngkDyE](https://user-images.githubusercontent.com/58752614/99891458-da2ae600-2c37-11eb-9236-6426f1ab8b47.gif)

Reply:
Due to performance costs when using `decimal.Parse`, isn't better to use and conditional to verify whatever the DisplayCount's length is greater than one and only then use TrimStart method?

Reply:
That's one way to approach it. However, there may be values like `0.0000123` etc.
If you can implement logic that handles the string representation of that, that's fine.
However, `decimal.TryParse` in my opinion is the way to go. There is also this issue open #32 so we will eventually be storing the decimal value instead of the string.

Reply:
Ok, I got it, but I only will make these changes tomorrow, the 22nd.

Btw, what do you mean by an "expression object"?

Reply:
By that I mean a new (POCO) class to simply store 2 decimal data type variables and the operator for the math expression.